frontend/src/components/QueryBuilder/Search/tag.tsx

```diff
@@ -0,0 +1,9 @@
+import { Tag as AntTag } from 'antd';
+import styled from 'styled-components';
+
+export const Tag = styled(AntTag)`
+	font-style: normal;
+	font-weight: 300;
+	font-size: 12px;
```

Comment:
rem

Reply:
it is with rem 
![image](https://user-images.githubusercontent.com/50579516/224322002-3e288466-4565-4c2e-a642-b2db2eae8027.png)